test/cpp/json-parse.cpp

```diff
@@ -0,0 +1,38 @@
+/*********************************************************************
+ * NAN - Native Abstractions for Node.js
+ *
+ * Copyright (c) 2017 NAN contributors
+ *
+ * MIT License <https://github.com/nodejs/nan/blob/master/LICENSE.md>
+ ********************************************************************/
+
+#include <nan.h>
+
+NAN_METHOD(Parse) {
+  Nan::JSON NanJSON;
+
+  Nan::MaybeLocal<v8::String> inp = Nan::To<v8::String>(info[0]);
+
+  if (inp.IsEmpty()) {
+    info.GetReturnValue().Set(Nan::Undefined());
```

Comment:
I replaced all `ReturnValue::Set(Nan::Undefined())` with `ReturnValue::SetUndefined()` ... You mentioned that the return value is `Undefined` by default, so there is actually no need to set any of these.   Since it was in a test suite I thought it best to be explicit.   Should I remove them or leave them as is, now that they've all been converted to `ReturnValue::SetUndefined()` ?